src/Common/src/System/Net/IPv6AddressHelper.Common.cs

```diff
@@ -411,6 +411,13 @@ internal static void Parse(ReadOnlySpan<char> address, Span<ushort> numbers, int
                 int toIndex = NumberOfLabels - 1;
                 int fromIndex = index - 1;
 
+                // if fromIndex and toIndex are the same 0 bits are already in the correct place
```

Comment:
does this mean `if fromIndex and toIndex are the same 0, then bits are already in the correct place` or something else? could be clearer.

Reply:
The meaning is that if from and to are same "0 bits"(compressed part) is already in correct place and doesn't need to moved up in the array position(called expanded in comment above) so maybe I need to place a comma before "0 bits".
Algorithm parse 16bits group in a sequence way and bookmark when a compressor(::) found.
If we have a compressor in the address means that at the end of parsed numbers we have the "0 compressor value" and we need to move in correct place up to arary index.
If calculation of fromIndex and ToIndex is the same mean that "0 value" is already in the correct position and we can return.

Reply:
I've updated the comment, let me know if it's more clear.